windows/src/main/java/com/github/weisj/darklaf/platform/windows/JNIDecorations.java

```diff
@@ -81,26 +80,26 @@ public static boolean updateLibrary() {
     }
 
     private static boolean loadLibrary() {
-        if (!SystemInfo.isWindows || !DarkLaf.isDecorationsEnabled()) {
+        if (!SystemInfo.isWindows) {
             return false;
         }
         if (loaded) return true;
         try {
             if (SystemInfo.isX86) {
-                NativeUtil.loadLibraryFromJar("/com/github/weisj/darklaf/platform/windows/x86/jniplatform.dll");
+                NativeUtil.loadLibraryFromJar("/com/github/weisj/darklaf/platform/darklaf-windows/windows-x86/libdarklaf-windows.dll");
```

Comment:
This should probably read ````darklaf-windows.dll```` instead of ````libdarklaf-windows.dll```` as that is the name of the generated library.

Reply:
Fixed, thanks